docs/ConfigureScalarDL.md

```diff
@@ -2,126 +2,43 @@
-  database: cosmos
-  contactPoints: <Cosmos DB account endpoint>
-  password: <Cosmos DB account primary master key>
-  cosmosBaseResourceUnit: 400
-```
-
-#### Configure scalardl-custom-values 
-
-To deploy Scalar DL Ledger on Cosmos DB, you need to update the [scalardl-custom-values.yaml](../conf/scalardl-custom-values.yaml) file.
-
-**With Kubernetes secrets**
-
-If you have created the `scalardl` Kubernetes secret, you can use the following configuration.
 
-```yaml
-ledger:
-  existingSecret: scalardl
-  ....
-  scalarLedgerConfiguration:
-    dbContactPoints: <Cosmos DB account endpoint>
-    dbStorage: cosmos
-```
-**Without Kubernetes secrets**
+## Reference Table
 
-If you have not created the `scalardl` Kubernetes secret, you can use the following configuration.
+You can refer to the table below to get values to create Kubernetes Secrets and update the configuration files for Scalar DL deployment according to the database you chose.
 
-```yaml
-ledger:
-  ....
-  scalarLedgerConfiguration:
-    dbContactPoints: <Cosmos DB account endpoint>
-    dbPassword: <Cosmos DB account primary master key>
-    dbStorage: cosmos
-```
+| Database  | storageType | contactPoints              | username       | password                                 | dynamoBaseResourceUnit | cosmosBaseResourceUnit | 
+|:----------|-------------|----------------------------|----------------|------------------------------------------|------------------------|------------------------|
+| DynamoDB  | dynamo      | AWS_REGION                 | AWS_ACCESS_KEY | AWS_ACCESS_SECRET_KEY                    | 10                     | N/A                    |
+| Cosmos DB | cosmos      | Cosmos DB account endpoint | N/A            | Cosmos DB account primary/secondary key  | N/A                    | 400                    |
+| JDBC      | jdbc        | JDBC_CONNECTION_URL        | USERNAME       | PASSWORD                                 | N/A                    | N/A                    |
 
-To enable Auditor service, you can follow the [Enable Auditor](#enable-auditor) section.
+Note:
 
-#### Configure scalardl-audit-custom-values
+* JDBC denotes all relational databases supported by Scalar DL such as MySQL, PostgreSQL, Oracle Database, SQL Server, and Amazon Aurora.
 
-To deploy the Scalar DL Auditor on Cosmos DB, you need to update the [scalardl-audit-custom-values.yaml](../conf/scalardl-audit-custom-values.yaml) file.
+## Create Kubernetes Secrets
 
-**With Kubernetes Secrets**
+Kubernetes Secret is an object that contains a small amount of sensitive data such as a password, a token, or a key.
+This optional method is recommended highly for handling credentials in the production environment.
+You can refer to the [reference table](#reference-table) section to get the values for `username` and `password` for the database you chose.
+You need to create a secret object with `db-username` and `db-password` to store the username and password of the underlying database.
```

Comment:
```suggestion
You need to create a secret object with key name `db-username` and `db-password` to store the username and password of the underlying database.
```
I think the `db-username` and `db-password` are the key name in the secret object.